go.mod

```
@@ -124,3 +124,5 @@ require (
 	modernc.org/memory v1.8.2 // indirect
 	modernc.org/sqlite v1.34.5 // indirect
 )
+
+replace github.com/doug-martin/goqu/v9 => github.com/ductone/goqu/v9 v9.19.1
```

golangci-lint (1.64.8)

replacement are not allowed: github.com/doug-martin/goqu/v9

(gomoddirectives)